test/common_test/Camera_TEST.cc

```diff
@@ -19,10 +19,13 @@
 
 #include "CommonRenderingTest.hh"
 
+#include <gz/utils/ExtraTestMacros.hh>
```

Comment:
Is this needed?

Reply:
Done https://github.com/gazebosim/gz-rendering/pull/755/commits/c8416ef049c9da229658e877fdf41af90517e486